src/java.base/share/classes/jdk/internal/classfile/impl/verifier/ParserVerifier.java

```diff
@@ -76,47 +75,44 @@ List<VerifyError> verify() {
-                }
-            };
-            check.accept(switch (cpe) {
-                case DoubleEntry de -> de::doubleValue;
-                case FloatEntry fe -> fe::floatValue;
-                case IntegerEntry ie -> ie::intValue;
-                case LongEntry le -> le::longValue;
-                case Utf8Entry ue -> ue::stringValue;
-                case ConstantDynamicEntry cde -> cde::asSymbol;
-                case InvokeDynamicEntry ide -> ide::asSymbol;
-                case ClassEntry ce -> ce::asSymbol;
-                case StringEntry se -> se::stringValue;
-                case MethodHandleEntry mhe -> mhe::asSymbol;
-                case MethodTypeEntry mte -> mte::asSymbol;
-                case FieldRefEntry fre -> {
-                    check.accept(fre.owner()::asSymbol);
-                    check.accept(fre::typeSymbol);
-                    yield () -> verifyFieldName(fre.name().stringValue());
-                }
-                case InterfaceMethodRefEntry imre -> {
-                    check.accept(imre.owner()::asSymbol);
-                    check.accept(imre::typeSymbol);
-                    yield () -> verifyMethodName(imre.name().stringValue());
-                }
-                case MethodRefEntry mre -> {
-                    check.accept(mre.owner()::asSymbol);
-                    check.accept(mre::typeSymbol);
-                    yield () -> verifyMethodName(mre.name().stringValue());
-                }
-                case ModuleEntry me -> me::asSymbol;
-                case NameAndTypeEntry nate -> {
-                    check.accept(nate.name()::stringValue);
-                    yield () -> nate.type().stringValue();
+            try {
+                switch (cpe) {
+                    case DoubleEntry de -> de.doubleValue();
+                    case FloatEntry fe -> fe.floatValue();
+                    case IntegerEntry ie -> ie.intValue();
+                    case LongEntry le -> le.longValue();
+                    case Utf8Entry ue -> ue.stringValue();
+                    case ConstantDynamicEntry cde -> cde.asSymbol();
+                    case InvokeDynamicEntry ide -> ide.asSymbol();
+                    case ClassEntry ce -> ce.asSymbol();
+                    case StringEntry se -> se.stringValue();
+                    case MethodHandleEntry mhe -> mhe.asSymbol();
+                    case MethodTypeEntry mte -> mte.asSymbol();
+                    case FieldRefEntry fre -> {
+                        fre.owner().asSymbol();
+                        fre.typeSymbol();
+                        verifyFieldName(fre.name().stringValue());
+                    }
+                    case InterfaceMethodRefEntry imre -> {
+                        imre.owner().asSymbol();
+                        imre.typeSymbol();
+                        verifyMethodName(imre.name().stringValue());
```

Comment:
With the new try-catch, say if all lines end up erroneous, we will only get 1 error instead of 3 like before.

Reply:
Fair point; given that there's only a couple of those though, maybe I could give them their own `try`/`catch`.